packages/next/export/index.js

```diff
@@ -52,7 +52,7 @@ export default async function (dir, options, configuration) {
 
   // Initialize the output directory
   const outDir = options.outdir
-  await del(join(outDir, '*'))
+  await rimraf(join(outDir, '*'), (e) => { throw e })
```

Comment:
This no longer returns a Promise, so it can't be await'd.

Reply:
Fixed :+1: